go/signedexchange/cmd/gen-signedexchange/main.go

```diff
@@ -30,6 +31,7 @@ func (h *headerArgs) Set(value string) error {
 
 var (
 	flagUri            = flag.String("uri", "https://example.com/index.html", "The URI of the resource represented in the exchange")
+	flagVersion        = flag.String("version", "1b2", "The signedexchange version")
```

Comment:
Let's use "1b1" as default until we have all the b2 changes in (i.e. http-mice-03).

Reply:
Agreed. Change this later then.